integrations/spark/openhouse-spark-runtime/build.gradle

```diff
@@ -4,6 +4,7 @@ plugins {
   id 'com.github.johnrengelman.shadow' version '6.0.0'
   id 'scala'
   id 'antlr'
+  id 'openhouse.iceberg-conventions'
```

Comment:
lets remove this, its dangerous for downstream client dependencies. Instead, pass the required dependencies as `--jars` argument to the spark-shell while spinning docker setup up

Reply:
Sounds good